ui/apps/platform/src/Containers/Dashboard/PatternFly/WidgetErrorEmptyState.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactNode } from 'react';
+import { Divider, Flex } from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import { SVGIconProps } from '@patternfly/react-icons/dist/js/createIcon';
+
+import EmptyStateTemplate from 'Components/PatternFly/EmptyStateTemplate';
+
+export type WidgetErrorEmptyStateProps = {
+    children: ReactNode;
+    height: `${number}px`;
+    title: string;
+};
+
+function ErrorIcon(props: SVGIconProps) {
+    return (
+        <ExclamationCircleIcon
+            {...props}
+            style={{ color: 'var(--pf-global--danger-color--200)' }}
+        />
+    );
+}
+
+export default function WidgetErrorEmptyState({
+    children,
+    title,
+    height,
+}: WidgetErrorEmptyStateProps) {
+    return (
+        <>
+            <Divider component="div" />
```

Comment:
Can you take a picture of an error state?

Because content is centered, I am trying to imagine whether the divider is necessary.

Reply:
Ack, I had these ready to go and never attached them to the PR. I've uploaded some screenshots of loading and error states.

I was torn for both cases on some style nits, that I can ping the UX team on:

For loading - whether or not to have padding around the grey skeleton component.
For error - whether or not to have the divider between the title.

Reply:
One other thought about error state. Unlike a table or form which displays nothing and offers few if any actions when request fails, for dashboard with several widgets which have only view actions (links, filtering, sorting) can you ask about pro and con of danger versus warning variant?